group_vars/shxco/vars.yml

```diff
@@ -22,13 +22,16 @@ clone_root: "/home/{{ deploy_user }}/repos"
 install_root: "/srv/www/{{ app_name }}"
 # apache location
 apache_app_path: "/var/www/{{ app_name }}"
+# use site name instead of 'mep' for apache site config
+apache_conf_name: "shakespeareandco"
 
+# allow any host, since we're behind PUL load balancer
+django_allowed_hosts:
+  - "*"
```

Comment:
i think we talked about moving this to the all vars, since only the new django role (and thus no RC apps) will use it

Reply:
weirdly it seems to be only defined for shxco_qa! do you think we vaulted it for other apps for some reason? 

I'll move it and check/remove settings for the other PUL deploys

Reply:
oh, duh — none of the other playbooks are using the django role yet. 

Would it make sense to create issues for some of the things we know about now? like updating playbooks to use the new django role, potentially moving the logging setup, etc.